data.php

```diff
@@ -222,6 +223,12 @@ function getBlockedQueries(\SplFileObject $log) {
         }
         return $lines;
     }
+
+    function countBlockedQueries() {
+        $hostname = trim(file_get_contents("/etc/hostname"), "\x00..\x1F");
+        return exec("grep \"gravity.list\" /var/log/pihole.log | grep -v \"pi.hole\" | grep -v -c \"".$hostname."\"");
```

Comment:
Add a `grep -v -c " read "` to the end (and remove the `-c` flag from the current last grep) to cover all the cases that the previous method covered.